### PR TITLE
Artifact Ranges Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Reduced long-range artifact effect max range from 200 tiles, to 20.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: html/changelogs/geeves-artifact_ranges.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - bugfix: "Reduced long-range artifact effect max range from 200 tiles, to 20."
```
Comment (Alberyk) [inline html/changelogs/geeves-artifact_ranges.yml]: Not a bugfix, because there is no bug to fix here.

Comment (Geevies) [reply html/changelogs/geeves-artifact_ranges.yml]: I don't agree, but alright.

Comment (Geevies) [inline html/changelogs/geeves-artifact_ranges.yml]:
```suggestion
  - tweak: "Reduced long-range artifact effect max range from 200 tiles, to 20."
```

**Pull request merged**